src/plugins/console/public/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -28,6 +28,9 @@ import { applyCurrentSettings } from './apply_editor_settings';
 const isJSONContentType = (contentType?: string) =>
   Boolean(contentType && contentType.indexOf('application/json') >= 0);
 
+const isMapboxVectorTile = (contentType?: string) =>
+  Boolean(contentType && contentType.indexOf('application/vnd.mapbox-vector-tile') >= 0);
```

Comment:
nit: why `includes` was changed to `indexOf`?

Reply:
what do you think about: 
`contentType?.includes('application/vnd.mapbox-vector-tile') ?? false`